### DISCUSSION TITLE
Theme selection

### DISCUSSION THREAD

**Discussion by TiagoDiass** [community/community#16177] [Other Feature Feedback, Questions, & Ideas] [answered]
I just wanted to suggest to you guys that you could create an easier way to change the current theme of the website.

The way to change the appearance theme nowadays is:

1. Click on your profile dropdown at the top right corner of the page
2. Click on the "Settings" option
3. Click on the "Appearance" option inside of the settings menu
4. Select your desired theme
5. Go back to the page you were in before wanting to change the theme preference.

The user experience could be improved if you guys implement an easier way to change the theme.
My suggestion is:

1. Click on your profile dropdown at the top right corner of the page that you're in
2. Click on a "Theme selection" option inside the dropdown
3. A modal is open for the user to select the theme
4. User selects a theme
5. Click on OK to close the modal, no needs to confirm the changes, as soon that user selects a theme, the page already updates its theme.

Example with images:

Dropdown with option
![image](https://user-images.githubusercontent.com/49597377/167234560-079b1fbf-8b78-4b63-9ec0-5f5b6cca91ba.png)

Theme selection modal
![image](https://user-images.githubusercontent.com/49597377/167235027-a77665f7-85d5-4869-b29d-1701f427f209.png)

Don't really take this images too seriously, I just did what I could changing the HTML elements of the site, they're just examples.

**Comment by fenilli** [comment, marked as answer]
Changing themes is mostly a one time thing, not necessarily meant to be changed many times if any at all after first setting, if we take this as a core idea for improvement, what would stop us from also adding a button for SSH keys, profile information and accessibility to the context menu, as those are changed as much or even more than themes.

I think this change would just clutter the context menu even more than it already is.
